services/supervisord/supervisord.go

```diff
@@ -697,6 +708,7 @@ command =
         {{- if .PerconaSSODetails}}
         cfg:default.server.domain="{{ .PMMServerAddress }}"
         cfg:default.auth.generic_oauth.enabled=true
+        cfg:default.auth.signout_redirect_url="https://{{ .IssuerDomain }}/login/signout?fromURI=https://{{ .PMMServerAddress }}/graph/login"
```

Comment:
Well, problem is we should support domains like http://localhost. Therefore, can we gran the protocol (http/https) from the PMM's url? My problem is PMM always assumes https< which is what breaks my PMM deployment on http://localhost. Someone tagged me today for the exact same problem.

Reply:
@atymchuk I think that is limitation of Okta. We have to use https. Am I right @maxkondr ?

Reply:
Okta rejects application domain with scheme != https.

Reply:
please see my comment https://jira.percona.com/browse/PMM-9287?focusedCommentId=292441&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-292441